pygbe/matrixfree.py

```diff
@@ -1368,7 +1368,7 @@ def dipole_moment(surf_array, electric_field):
     attribute of the surface class. The dipole is expressed as a boundary
     integral.
 
-    Arguments:
+    Arguments
     ----------
```

Comment:
Remove one of the `-` here. Sphinx will throw a warning if the length of the `-`s doesn't match the text above